Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -472,12 +472,16 @@ export async function unlockKeychain(keychainPath: string, keychainPwd: string):
  * @param uuid
  */
 export async function deleteProvisioningProfile(uuid: string): Promise<void> {
-    let provProfilePath: string = getProvisioningProfilePath(uuid);
-    tl.warning('Deleting provisioning profile: ' + provProfilePath);
-    if (tl.exist(provProfilePath)) {
-        let deleteProfileCommand: ToolRunner = tl.tool(tl.which('rm', true));
-        deleteProfileCommand.arg(['-f', provProfilePath]);
-        await deleteProfileCommand.exec();
+    let provProfiles: string[] = tl.findMatch(getUserProvisioningProfilesPath(), uuid + '*');
```

Comment:
```suggestion
    let provProfiles: string[] = tl.findMatch(getUserProvisioningProfilesPath(), uuid.trim() + '.*');
```

`.trim()` for consistency with getProvisioningProfilePath, and `.` to match on exactly the uuid.